### ISSUE TITLE
Support ISPC language

### ISSUE THREAD

**Issue by dbabokin** [ccache/ccache#847]
ISPC language (https://ispc.github.io/) is used in a number of rendering and game dev projects. Some of the projects are fairly large and would benefit from `ccache` support for ISPC. We have specific request from Unreal Engine 4 user to have such support (see here: https://github.com/ispc/ispc/discussions/1944#discussioncomment-709253). I'm sure other project will benefit from that as well.

ISPC command line switches are not a derivative from clang or any other compiler. But it's a relatively short list of switches.

Would you accept a patch that supports ISPC? I'm willing to try implementing it, but would like to check first if there's a chance for merging such patch.

**Comment by jrosdahl** [comment]
Yes, support for ISPC could be accepted if it's not too intrusive or complex.
